### PR TITLE
My branch here

### PR DESCRIPTION
**Pull request by abhishekpatel946**
Hey, @mycrazyfish
 
I'm added an api/README.md. 

> if any changes want to in README.md please let me know

### CODE REVIEW

File: api/README.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+- add features
+- add how it works
+- add some cool features
+
+
+### Built With
+
+This section should list any major frameworks that you built your project using. Leave any add-ons/plugins for the acknowledgements section. Here are a few examples.
+
+- [JavaScript](https://www.javascript.com/)
```
Comment (mycrazyfish) [inline api/README.md:53]: lets keep this JS only for now, remove the others. we will get contibutors to add this as they build it.

File: api/README.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+### Installation
+
+1. Clone the repo
+```sh
+git clone https://github.com/mycrazyfish/nodesql
+```
+
+2. Use the package manager npm to install and run this API.
+```sh
```
Comment (mycrazyfish) [inline api/README.md:73]: cd to api?

File: api/README.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+3. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
+4. Push to the Branch (`git push origin feature/AmazingFeature`)
+5. Open a Pull Request
+
+<!-- LICENSE -->
+
+## License
+
+<!-- Here you can add license link  -->
+Distributed under the License. See [`LICENSE`](https://github.com/mycrazyfish/nodesql) for more information.
```
Comment (mycrazyfish) [inline api/README.md:121]: Lets us the main repo MIT Licence

**Review by mycrazyfish — COMMENTED** [review]
lets just tidy up a tiny bit? really appreciate your help. looks awesome.

File: api/README.md (hunk 2 above)
Comment (abhishekpatel946) [reply api/README.md:73]: okey! got it

File: api/README.md (hunk 1 above)
Comment (abhishekpatel946) [reply api/README.md:53]: ok!

File: api/README.md (hunk 3 above)
Comment (abhishekpatel946) [reply api/README.md]: Okey! 
it means to put the link on 
**LICENSE** `https://choosealicense.com/licenses/mit/`
from the main repo

File: api/README.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![GitHub license](https://img.shields.io/github/license/mycrazyfish/nodesql)](https://github.com/mycrazyfish/nodesql)
+
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/abhishekpatel946/Budget-Manager-app">
```
Comment (mycrazyfish) [inline api/README.md:10]: fix the href here

**Review by mycrazyfish — COMMENTED** [review]
missed href on my first review

**Comment by abhishekpatel946** [comment]
This is an updated README.md and changes according to your need.

**Comment by mycrazyfish** [comment]
@abhishekpatel946 if you could fix up the href for the logo, i will then merge this

**Comment by abhishekpatel946** [comment]
This changes is good be to merge

**Review by mycrazyfish — APPROVED** [review]

**Comment by abhishekpatel946** [comment]
Thank you @mycrazyfish 🙂 for your step by step guidance. Its my first PR✨ and first time I contribute on some else's code.

If you want more help amd collab. Feel free to reach me✨✨

**Comment by mycrazyfish** [comment]
> Thank you @mycrazyfish 🙂 for your step by step guidance. Its my first PR✨ and first time I contribute on some else's code.
> 
> If you want more help amd collab. Feel free to reach me✨✨

thanks for your contib, lets make the world a better place with opensource. cheers

**Pull request merged**